pilot/pkg/networking/grpcgen/grpcgen_test.go

```diff
@@ -163,7 +164,7 @@ func TestGRPC(t *testing.T) {
 		rb := xdsresolver
 		stateCh := &Channel{ch: make(chan interface{}, 1)}
 		errorCh := &Channel{ch: make(chan interface{}, 1)}
-		_, err := rb.Build(resolver.Target{Endpoint: istiodSvcAddr},
+		_, err := rb.Build(resolver.Target{URL: url.URL{Scheme: "xds", Path: "/" + istiodSvcAddr}},
```

Comment:
@stevenctl I couldn't figure out what change broke us. Not sure if it impacts real usages or just our low level test

Reply:
Was this a build failure or runtime failure?

Reply:
Its a runtime failure. Basically without it we get an LDS request for `""`. Interestingly, the code here works on 1.43 but does NOT work on 1.42.